extensions/twitter/README.md

```diff
@@ -0,0 +1,36 @@
+# Twitter extension
+This document defines how to describe twitter extension on AsyncAPI.
+
+## Twitter Object
+The twitter object must contains  : 
+
+Field Name | Type | Description
+---|:---:|---
+<a name="extensionId"></a>`id` | string | **Required**. Id of the extension.
+<a name="extensionTitle"></a>`title` | string |**Required**. The title of the extension. 
+<a name="extensionDescription"></a>`description` |string| The description of the twitter extension. 
+<a name="extnesionPayload"></a>`payload` | Json Object  |**Required**. A JSON object containing the detail of the twitter account. 
```

Comment:
Why don't we simply merge this object with the `payload` one? We could just add `schema` and `examples` right into this object.